exporter/opentelemetry-exporter-zipkin/src/opentelemetry/exporter/zipkin/__init__.py

```diff
@@ -96,7 +101,7 @@ def __init__(
         local_node_ipv4: IpInput = None,
         local_node_ipv6: IpInput = None,
         local_node_port: Optional[int] = None,
-        max_tag_value_length: Optional[int] = None,
+        max_tag_value_length: Optional[int] = DEFAULT_MAX_TAG_VALUE_LENGTH,
```

Comment:
If I remember it correctly this default `None` disables the tag value truncation feature. Are we changing that behaviour?

Reply:
Agree. Should revert or document in changelog.

Reply:
Is that the expected behavior? When testing this (before the change), I did not set `max_tag_value_length` at all and it actually crashes the application. The arg is [passed explicitly](https://github.com/open-telemetry/opentelemetry-python/blob/main/exporter/opentelemetry-exporter-zipkin/src/opentelemetry/exporter/zipkin/__init__.py#L116) and encoder [sets the value](https://github.com/open-telemetry/opentelemetry-python/blob/main/exporter/opentelemetry-exporter-zipkin/src/opentelemetry/exporter/zipkin/encoder/__init__.py#L70) so it winds up being `None`, which winds up crashing [here](https://github.com/open-telemetry/opentelemetry-python/blob/main/exporter/opentelemetry-exporter-zipkin/src/opentelemetry/exporter/zipkin/encoder/__init__.py#L147). Does `ZipkinExporter` expect an explicitly passed in value for `max_tag_value_length`?

Reply:
That's a good catch. From what I can see Jeager has null checks for `max_tag_value_length`  but Zipkin doesn't. We can be consistent with either by adding None checks to Zipkin or update the Jeager to also default `DEFAULT_MAX_TAG_VALUE_LENGTH`. I would prefer updating Zipkin to handle None because some users might not want tag value truncation.